Projects/Server/Utilities/Utility.cs

```diff
@@ -560,31 +561,10 @@ public static string FixHtml(string str)
                 return "";
             }
 
-            var hasOpen = str.ContainsOrdinal('<');
-            var hasClose = str.ContainsOrdinal('>');
-            var hasPound = str.ContainsOrdinal('#');
-
-            if (!hasOpen && !hasClose && !hasPound)
-            {
-                return str;
-            }
-
-            var sb = new StringBuilder(str);
-
-            if (hasOpen)
-            {
-                sb.Replace('<', '(');
-            }
-
-            if (hasClose)
-            {
-                sb.Replace('>', ')');
-            }
-
-            if (hasPound)
-            {
-                sb.Replace('#', '-');
-            }
+            using var sb = new ValueStringBuilder(str, stackalloc char[Math.Min(40960, str.Length)]);
```

Comment:
This is a stupid hack. I have to supply a stackalloc, otherwise I can't use stackalloc invalid/replacement char arrays. It gives me a leaky context error. ugh